cmd/root.go

```diff
@@ -165,6 +174,9 @@ var diffCmd = &cobra.Command{
 			color.Red(err.Error())
 			os.Exit(1)
 		}
+		if client.Errors > 0 {
+			os.Exit(1)
+		}
```

Comment:
Add this to the other places where we can continue-on-err

Reply:
I already did this actually. I was confused because the `reckoner.NewClient` function requires it as an arg even on commands that don't make use of that functionality.